zeroex/orderwatch/order_watcher.go

```diff
@@ -367,6 +367,74 @@ func (w *Watcher) handleBlockEvents(events []*blockwatch.Event) error {
 					return err
 				}
 
+			case "ERC1155TransferSingleEvent":
+				var transferEvent decoder.ERC1155TransferSingleEvent
+				err = w.eventDecoder.Decode(log, &transferEvent)
+				if err != nil {
+					if isNonCritical := w.checkDecodeErr(err, eventType); isNonCritical {
+						continue
+					}
+					return err
+				}
+				// HACK(fabio): Currently we simply revalidate all orders involving assets in this
```

Comment:
Are we sure this stuff regarding Augur is going to be true anymore?

Reply:
Yes, since Augur will be using a DummyERC1155 token contract and the actual settlement could involve a slew of "complete set" magic, we cannot rely on the "ID" as we could with a regular ERC1155 token.